.gitattributes

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 The Khronos Group Inc.
+# Copyright 2023 Valve Corporation
+# Copyright 2023 LunarG, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# See https://git-scm.com/docs/gitattributes
+# See https://help.github.com/articles/dealing-with-line-endings/
+
+# Default behavior, if core.autocrlf is unset.
+* text=auto
+
+# Files to be converted to native line endings on checkout.
+*.cpp text
+*.h text
+
+# Public header files will always have LF (unix) line endings on checkout.
+vk_*.h text eol=lf
+vk_*.hpp text eol=lf
+
+# Text files to always have CRLF (dos) line endings on checkout.
+*.bat text eol=crlf
```

Comment:
Are there .bat & .sh files in this repo? Not against including them here, since they *are* platform specific files but just wanted to check if this is intentional.

Reply:
Nope there aren't any. I'll remove that bit.